micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/ExecutorServiceMetrics.java

```diff
@@ -209,6 +210,96 @@ private void monitor(MeterRegistry registry, @Nullable ThreadPoolExecutor tp) {
                 .description("The current number of threads in the pool")
                 .baseUnit(BaseUnits.THREADS)
                 .register(registry);
+
+        monitorRejectedExecutionHandler(registry, tp);
+        monitorThreadFactory(registry, tp);
+    }
+
+    private void monitorThreadFactory(MeterRegistry registry, ThreadPoolExecutor tp) {
+        MonitoredThreadFactory tf = new MonitoredThreadFactory(tp.getThreadFactory());
+        tp.setThreadFactory(tf);
+
+        Gauge.builder("thread.factory.created", tf, MonitoredThreadFactory::getCreated)
```

Comment:
This sounds like a monotonically increasing number. If so, it should be a `FunctionCounter` instead of a `Gauge`.